Templates/HttpPUT(CRUD)-CSharp/run.csx

```diff
@@ -0,0 +1,31 @@
+﻿#r "Microsoft.WindowsAzure.Storage"
+
+using System.Net;
+using Microsoft.WindowsAzure.Storage.Table;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, CloudTable outTable, TraceWriter log)
+{
+    dynamic data = await req.Content.ReadAsAsync<object>();
+    string name = data?.name;
+
+    if (name == null)
+    {
+        return req.CreateResponse(HttpStatusCode.BadRequest, "Please pass a name in the request body");
+    }        
+
+    Person updatedPerson = new Person()
+    {
+        PartitionKey = "Functions",
+        RowKey = Guid.NewGuid().ToString(), // or guid for existing row key
+        Name = name
+    };
+    
+    TableOperation updateOperation = TableOperation.InsertOrReplace(updatedPerson);
+    outTable.Execute(updateOperation); 
+    return req.CreateResponse(HttpStatusCode.Created);
```

Comment:
Probably should only return a 201 created status if the entity was created, otherwise return a 200 accepted for the update?